### PR TITLE
Fixes for LVS export and import

### PR DESCRIPTION
**Pull request by dsavitskiy**

### CODE REVIEW

**Review by hrudaya21 — APPROVED** [review]

**Review by tiagolobocastro — APPROVED** [review]
Linting is failing, please fix that before merging.

Btw this is a chunky change, I wonder if we should attempt to merge this upstream?

File: lib/blob/request.c [hunk 1]
```diff
@@ -14,6 +14,107 @@
...
+	if (!ps) {
+		return -ENOMEM;
+	}
+	req_pool->req_mem = ps;
+
+	// Final NULL element.
+	req_pool->req_mem[i] = NULL;
+
+	// Allocate a new chunk of spdk_bs_request_set elements.
+	req_pool->req_mem[i - 1] = set = calloc(max_ops, sizeof(struct spdk_bs_request_set));
```
[the remark this answers is not in the document]
Comment (dsavitskiy) [reply lib/blob/request.c:69]: Why? What would it change?

**Comment by dsavitskiy** [comment]
> Linting is failing, please fix that before merging.
> 
> Btw this is a chunky change, I wonder if we should attempt to merge this upstream?

Yes, I think we should.

File: lib/blob/request.c (hunk 1 above)
[the remark this answers is not in the document]
Comment (tiagolobocastro) [reply lib/blob/request.c:69]: We don't need to grow the req_mem if we couldn't allocate the request?

[the remark this answers is not in the document]
Comment (dsavitskiy) [reply lib/blob/request.c:69]: Probably yes, the code will look cleaner. (But it would still fail anyway.)

**Pull request merged**